### PR TITLE
Two definitions: affine schemes, coupled spaces

### PR DESCRIPTION
**Pull request by felixwellen**
Definitions and related propositions. Most notably, that the sigma type of algebras with spectrum isomorphic to a fixed coupled space is a proposition.

### CODE REVIEW

File: SyntheticGeometry/SQC.lagda.md [hunk 1]
```diff
@@ -93,10 +141,10 @@ But even more, every nonzero vector contains an invertible element.
       finite-presentation-of-A : FinitePresentation A
       finite-presentation-of-A = Instances.R/⟨xs⟩FP k xs
 
-      equiv : ⟨ A ⟩ ≃ (Spec k A → ⟨ k ⟩)
+      equiv : _ ≃ (Spec A → _)
```
Comment (MatthiasHu) [inline SyntheticGeometry/SQC.lagda.md:144]: What was the intention behind reducing the type signatures around here?

File: SyntheticGeometry/Affine.lagda.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+```
+
+Finite affine qc-open covers
+----------------------------
+The following is used to define qc-schemes:
+
+```agda
+
+is-affine-finite-qc-open-cover : {n : ℕ}
```
Comment (MatthiasHu) [inline SyntheticGeometry/Affine.lagda.md:200]: I feel like this should be in `qc-Scheme.lagda.md`. Are there reasons to have it here?

**Review by MatthiasHu — CHANGES_REQUESTED** [review]
Yay! :-)

File: SyntheticGeometry/SQC.lagda.md (hunk 1 above)
Comment (felixwellen) [reply SyntheticGeometry/SQC.lagda.md:144]: It was about avoiding this error:
```
Instance declarations with explicit arguments are never considered
by instance search, so making SyntheticGeometry.SQC._ into an
instance has no effect.
when checking that A is a valid argument to a function of type
{ℓᵢ ℓc : Level} {Instance : Type ℓᵢ}
⦃ r : Cubical.Syntax.⟨⟩.has-⟨⟩ Instance ⦄ →
Instance → Type ℓc
```

Comment (felixwellen) [reply SyntheticGeometry/SQC.lagda.md:144]: We could also just live with the warning or figure out, why it comes up here. I don't understand, how we are making anything into an instance (claimed by the warning).

Comment (felixwellen) [reply SyntheticGeometry/SQC.lagda.md:144]: Or I just finally fix this part of cubical, i.e. remove the instances and just replace it with projection.

File: SyntheticGeometry/Affine.lagda.md (hunk 2 above)
Comment (felixwellen) [reply SyntheticGeometry/Affine.lagda.md]: Moved

File: SyntheticGeometry/SQC.lagda.md (hunk 1 above)
Comment (felixwellen) [reply SyntheticGeometry/SQC.lagda.md:144]: I made a PR (#903).

Comment (MatthiasHu) [reply SyntheticGeometry/SQC.lagda.md:144]: Do you mean https://github.com/agda/cubical/pull/907?

Comment (felixwellen) [reply SyntheticGeometry/SQC.lagda.md]: Yes. It is merged now.

**Review by MatthiasHu — APPROVED** [review]

**Pull request merged**